tests/file_descriptor.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+IFS='
+';
+INITIAL_FILE_DESCRIPTOR_STORE=${INITIAL_FILE_DESCRIPTOR_STORE:-"initial.txt"}
+CURRENT_FILE_DESCRIPTOR_STORE=${CURRENT_FILE_DESCRIPTOR_STORE:-"final.txt"}
+
+declare -A initial
+
+while IFS="" read -r p || [ -n "$p" ]
+do
+  SERVICE=$(echo $p | cut -d" " -f 1)
+  COUNT_DESCRIPTORS=`echo $p | cut -d" " -f 2`
```

Comment:
Codacy found an issue: [Use $(...) notation instead of legacy backticked `...`.](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5973213)